src/app/inventory/store/d2-item-factory.ts

```diff
@@ -1,4 +1,6 @@
+import { D1ManifestDefinitions } from 'app/destiny1/d1-definitions';
```

Comment:
does d1 still have trials?

Reply:
wait more importantly this is a d2 item building file. it's impossible for the definitions to be d1 definitions

Reply:
Yep you're right. I can change the reference in `ItemDetails.tsx` that uses D1 and check it's a D2 def first like this:

`isTrialsPassage={defs.isDestiny2() && isTrialsPassage(item, defs)}`

Reply:
ahh, i see now it's exported.
yeah if it's a more general helper function, definitely find a nice place to put it that's not d2 item factory file.